test/process-results.py

```diff
@@ -8,6 +8,7 @@
 from functools import reduce
 from collections import OrderedDict, defaultdict
 from datetime import datetime, timedelta
+from html import escape as html_escape
```

Comment:
I have a slight preference for using `import html` and then calling the function via `html.escape`, but I see that I already used `html` as a variable name. Perhaps a future refactoring could fix that.

Reply:
Yeah, same here, but I didn't want to explode the diff. That module only contains two functions, so I was fine with this.